native/packages/react-native-bpk-component-button/readme.md

```diff
@@ -56,6 +56,58 @@ export default class App extends Component {
 }
 ```
 
+### With Theming
+
+```js
+import React, { Component } from 'react';
+import { View } from 'react-native';
+import BpkButton from 'react-native-bpk-component-button';
+import BpkThemeProvider from 'react-native-bpk-theming';
+import * as TOKENS from 'bpk-tokens/tokens/ios/base.react.native.es6';
+
+import { translationHelper } from 'translations';
+
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    justifyContent: 'center',
+    padding: TOKENS.spacingBase,
+  }
```

Comment:
es-lint will complain about a missing trailing comma!

Reply:
Not in this case as it's a readme but good spot!

Reply:
I mean if a user copy-pastes it 👍

Reply:
Fixed!